src/main/java/com/contrastsecurity/http/SecurityCheckForm.java

```diff
@@ -52,12 +49,31 @@
     @SerializedName("origin")
     private String origin = "OTHER";
 
-    public SecurityCheckForm(String applicationId) {
+    /**
+     * The job start time of the jenkins job
```

Comment:
Can you add a sentence here that some job outcome policy configurations need the jobStartTime?